core/spec/models/spree/promotion_spec.rb

```diff
@@ -384,6 +384,73 @@
     end
   end
 
+  context '#unstarted?' do
+    it 'should not be unstarted if no starts_at date' do
```

Comment:
(more hair splitting 💇‍♂️) in my experience using `should` is a bit out of fashion with rspec. [This link](http://www.betterspecs.org/#should) may be one of the reasons, together with the new `expect` syntax that replaced `should`. 
Again, I understand this file is already full of old code that uses `should` and if we want to be consistent we should keep using it...